### DISCUSSION TITLE
Vertical scroll on drawer menu

### DISCUSSION THREAD

**Discussion by johngouf** [saadeghi/daisyui#1319] [Q&A] [answered]
Hi! First of all kudos for this amazing library!
Was using the drawer component and at the same time I had the developer tools opened, so I noticed that the menu items get split to 2 columns. Is it possible  instead to have a vertical scroll?
Attaching an image to show the behaviour:
![image](https://user-images.githubusercontent.com/1265084/200055977-380e273f-0741-4adf-92f6-77d088e57e24.png)

Thanks!

**Comment by saadeghi** [comment, marked as answer]
Remove the `overflow-y-auto` class.  
I see that it's on the docs examples by mistake. My bad. Fixed it.

**Comment by johngouf** [comment]
Thanks so much for the prompt response! Works great now!
